src/plugins/npm/npmPlugin.js

```diff
@@ -1,3 +1,4 @@
+const packageNameRegex = require('package-name-regex');
```

Comment:
if we're going to use a package for it, lets use `npm-package-arg`, since that's actually the one npm uses

Reply:
Ok good point, done